pkg/storage/util.go

```diff
@@ -49,3 +50,11 @@ func ParseWatchResourceVersion(resourceVersion, kind string) (uint64, error) {
 	}
 	return version + 1, nil
 }
+
+func NamespaceKeyFunc(prefix string, obj runtime.Object) (string, error) {
```

Comment:
It's neither move nor duplication.
There is a similar method under pkg/registry/generic/etcd/etcd.go, but that method is taking data from api.Context. Here we work just on runtime.Object (and we get all the data from there).